index.html

```diff
@@ -224,6 +229,21 @@
       <p>To simplify style authoring, [[!ttml-imsc1.1]] Text Profile SHALL support the <code>#initial</code> feature specified in
       [[!TTML2]].</p>
     </section>
+
+    <section>
+      <h3><code>#textShadow</code></h3>
+
+      <p>To support the edge styles required by the FCC, [[!ttml-imsc1.1]] Text Profile SHALL support the <code>#textShadow</code>
```

Comment:
Is this genuinely a requirement of FCC that is not currently met with textOutline? I thought that had already been deemed sufficient.

Reply:
@mikedo What say you?

Reply:
There are some issues with tts:textOutline but I can not remember what they have been. Possibly we can look at some examples.

Reply:
> There are some issues with tts:textOutline but I can not remember what they have been. Possibly we can look at some examples.
 
Per the joint CSS/TTWG meeting, text-outline is not well-supported in UAs today. No one disputed the benefit of the feature however.

Reply:
Yes, that's what I recall - the problem with `tts:textOutline` is not that it is a poor fit for the requirement, but that it is not widely implemented. If it _were_ implemented, presumably support for `#textShadow` would no longer be required to support the edge styles required by FCC?